Purchases/src/main/java/com/theironyard/PurchasesController.java

```diff
@@ -0,0 +1,72 @@
+package com.theironyard;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.annotation.PostConstruct;
+import javax.servlet.http.HttpSession;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Created by EddyJ on 7/20/16.
+ */
+@Controller
+public class PurchasesController {
+
+    @Autowired
+    CustomerRepository customers;
+
+    @Autowired
+    PurchaseRepository purchases;
+
+    @PostConstruct
+    public void init() throws FileNotFoundException {
```

Comment:
A little bit of a cheat as this will incinerate the program in the event that an exception happens which could cause an issue if something swallows it.